<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drop_courses</class>
 <widget class="QMainWindow" name="drop_courses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drop Courses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="drop_courses_table_widget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>811</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instructor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="drop_courses_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Drop Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drop_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>580</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
